### PR TITLE
Added automatic per-app language support

### LINKED ISSUE #2662: System setting per-app language API
I want to add the ability to change the app language in Android system settings using the per-app language settings - [https://developer.android.com/guide/topics/resources/app-languages](https://developer.android.com/guide/topics/resources/app-languages).
Just "System settings" without additional APIs inside the app.

### PR DESCRIPTION
**Pull request by jedi1150**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [✅] The PR is submitted to the `main` branch.
- [✅] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).
- [✅] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [✅] My code builds and is tested on a real Android device.
- [✅] I confirm that I've run the code locally and everything works as expected.

## What's changed?

Screenshot 1
---
photo_2023-09-22 01 17 39

screen recording
---
https://github.com/Ivy-Apps/ivy-wallet/assets/15169545/af1691e0-9b4b-4001-a484-c16535a6465a

Describe with a few bullets **what's new:**
- Ability to change the app language through the Android system settings.

_💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!_

## Risk Factors

**What may go wrong if we merge your PR?**

- Untranslated string resources
- Strings in data classes annotated as `@Immutable`

**In what cases your code won't work?**

- Android 12 and less

## Does this PR closes any GitHub Issues?

- Closes #2662

### CODE REVIEW

File: app/src/main/res/resources.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+unqualifiedResLocale=en-US
```
Comment (ILIYANGERMANOV) [inline app/src/main/res/resources.properties:1]: What does that line mean? Is this the default app language?

**Review by ILIYANGERMANOV — APPROVED** [review]
LGTM! Can you confirm that the default app language remain English? Thank you for your contribution 🎉 It would be also awesome if we there's a way to add such setting in Ivy Wallet's Settings screen, too. But this PR is good and we can merge it after the CI pass ✅

**Comment by ILIYANGERMANOV** [comment]
@jedi1150 very well written PR description btw 💯 👏 Thanks for adding the Risk Factors

File: app/src/main/res/resources.properties (hunk 1 above)
Comment (jedi1150) [reply app/src/main/res/resources.properties:1]: Yes, this is the default localization. It's a required parameter.

Comment (ILIYANGERMANOV) [reply app/src/main/res/resources.properties:1]: Got it. Thanks for the explanation!

**Pull request merged**